ws-server/src/server.rs

```diff
@@ -404,9 +404,50 @@ impl Builder {
 
 	/// Register a new resource kind. Errors if `label` is already registered, or if number of
 	/// registered resources would exceed 8.
+	///
```

Comment:
The thing that worries me about these duplicated docs is keeping them both in sync in the event that things change. I don't _think_ that there is a nice way to share documentation though is there?

I'd be tempted to have one version of the docs link to the other version eg

```
/// Register a new reousrce kind. See [`crate::heep_server::Builder::register_resource()`] for example usage.
```

Reply:
> I don't _think_ that there is a nice way to share documentation though is there?

I don't know of a way either and it's quite common to see "look over here for details" in rustdocs.

Reply:
I still have the duplicated comments on these, but the bulk of it has been moved to the `resource_limiting` module, link included, so I reckon this is resolved now?